scripts/release.sh

```diff
@@ -59,43 +60,178 @@ function release_app {
     echo "_____________________________________________"
```

Comment:
I always think that this is not the way to flag the "latest" image, now that you check the different VERSION values and branches we could find out if the current version number is really the latest one. (Obviously in another PR)

```bash
    if [[ $VERSION != "alpha" ]]
    then
        echo "Pushing Docker image ${IMAGE_REPO}/${AETHER_APP}:latest"
        docker tag ${AETHER_APP} "${IMAGE_REPO}/${AETHER_APP}:latest"
        docker push "${IMAGE_REPO}/${AETHER_APP}:latest"
    fi
```

My concern:

We have `release-1.0`, ...`release-2.1` branches. We tagged `1.7.4`. With this condition the release `1.7.4`  is going to be the `latest` but there is already a `2.0.0` one... :woman_shrugging:

Reply:
Are you going to remove this piece of code?

Reply:
@lordmallam ?

Reply:
@obdulia-losantos Yes, removed